config/locales/curation_concerns.en.yml

```diff
@@ -159,6 +159,8 @@ en:
   file_manager:
     link_text: 'File Manager'
   simple_form:
+    required:
+      html: '<span class="label label-info required-tag">required</span>'
```

Comment:
Can we move the html into the simple_form initializer? That wouldn't change even if we're using a different language.